packages/components/src/Layout/Semantics/Semantics.test.tsx

```diff
@@ -108,4 +109,47 @@ describe('Layout', () => {
       'overflow: hidden;'
     )
   })
+  test('using prop scrollWithin will have areas scrolling together.', () => {
+    renderWithTheme(
+      <Page fixed>
+        <Layout hasAside>
+          <Aside scrollWithin>I'm the Aside before</Aside>
+          <Section scrollWithin>I'm the main area</Section>
+          <Aside scrollWithin>I'm the Aside after</Aside>
+        </Layout>
+      </Page>
+    )
+    expect(screen.getByText("I'm the Aside before")).toHaveStyle(
+      'height: fit-content;'
+    )
+    expect(screen.getByText("I'm the main area")).toHaveStyle(
+      'height: fit-content;'
+    )
+    expect(screen.getByText("I'm the Aside after")).toHaveStyle(
+      'height: fit-content;'
+    )
+  })
+
+  xtest('using prop scrollWithin will have only selected areas scrolling together.', () => {
+    renderWithTheme(
+      <Page fixed>
+        <Layout hasAside>
+          <Aside>I'm the Aside before</Aside>
+          <Layout hasAside>
+            <Section scrollWithin>I'm the main area</Section>
+            <Aside scrollWithin>I'm the Aside after</Aside>
+          </Layout>
+        </Layout>
+      </Page>
+    )
+    expect(screen.getByText("I'm the Aside before")).not.toHaveStyle({
+      height: 'fit-content',
+    })
```

Comment:
this passes if you change to `toHaveStyleRule`

```suggestion
    expect(screen.getByText("I'm the Aside before")).not.toHaveStyleRule(
      'height'
    )
```